index.html

```diff
@@ -5,15 +5,58 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./css/fonts.css">
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+<header>
+    <img src="./img/karma-logo.svg" width="10" height="10" alt="karma" id="karma">
+<nav class="nav">
+   <a href="https://stackoverflow.com/questions/3014469/what-is-the-standard-way-to-add-an-icon-to-a-link-with-css" class="nav__link">Meet Karma</a>
```

Comment:
we don;t need those long links in nav bar, 
you can just simply have `<a href="#">....</a>`